scripts/validate-grafana-dashboard.js

```diff
@@ -0,0 +1,6 @@
+let lodestarDash = require("../docker/grafana/provisioning/dashboards/lodestar.json");
+lodestarDash = JSON.stringify(lodestarDash); //get everything in the same line
```

Comment:
Here you are parsing a JSON file to immediately serialize it again. You can just read it with fs.readFileSync which is what require() is doing under the hood.

Reply:
yes, doing it intentionally to remove  bring it back to same liine, reason it could be:
exemplar :
    true

also side benefit: it is sort of check that JSON is actually parse-able .

Reply:
Oh! Good point, then sounds good to me